packages/editor-ui/src/components/CodeNodeEditor/theme.ts

```diff
@@ -46,7 +50,10 @@ export const CODE_NODE_EDITOR_THEME = [
 			borderLeftColor: 'var(--color-code-caret)',
 		},
 		'&.cm-focused .cm-selectionBackgroundm .cm-selectionBackground, .cm-content ::selection': {
-			backgroundColor: 'var(--color-code-selection)',
+			// backgroundColor: 'var(--color-code-selection)',
```

Comment:
for some reason background-color on ::selection makes selection transparent on chrome (on linux). that's why I had to disable this. but, I'll revert this now, as the bug doesn't seem to exist on other platforms.